_posts/en/newsletters/2020-10-21-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #120'
+permalink: /en/newsletters/2020/10/21/
+name: 2020-10-21-newsletter
+slug: 2020-10-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter provides an overview of the new MuSig2 paper,
+summarizes additional discussion about upfront fees in LN, and describes
+a proposal to simplify management of LN payments.  Also included are our
+regular sections with summaries of notable improvements to clients and
+services, announcements of releases and release candidates, and changes
+to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- MuSig2 FIXME:jnewbery
+
+- **More LN upfront fees discussion:** after [last week's
+  discussion][news119 upfront], Joost Jager asked developers on the
+  Lightning-Dev mailing list to again [consider][jager hold fees] how to
+  charge fees for attempting to route payments---fees that would have to
+  be paid even if the attempt failed.  This could mitigate jamming
+  attacks that can prevent nodes from earning routing fees.
```

Comment:
It seems to me that making network probing attacks (e.g. for tracking channel balances) more costly is also a central concern that is addressed by this proposal.